os/board/rtl8730e/src/component/bluetooth/driver/platform/amebad2/hci/hci_platform.c

```diff
@@ -972,7 +972,6 @@ static uint8_t* hci_platform_get_btfw_patch(uint32_t *rtlbt_fw_size)
     /* Below condition is minimal requirement to meet */
 	/* Address must be starting of the external flash, size shouldn't be larger than 128KB */
     if(!((*rtlbt_fw_size > 0 && *rtlbt_fw_size < 0x20000) && (rtlbt_fw_addr == EXT_FLASH_BASE_ADDRESS))) {
-        dbg("Please check whether RTK data binary is flashed correctly!");
```

Comment:
Hello @jeongarmy , as the external download is ready now. I'm going to remove this line now, do you think is it okay?